src/cli.py

```diff
@@ -2,27 +2,48 @@
 
 import argparse
 
-import menu_parser
+import menu_parser  # pylint: disable=import-error
 
 
 def parse_cli_args():
     parser: argparse.ArgumentParser = argparse.ArgumentParser()
-    group: argparse._MutuallyExclusiveGroup = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument("-p", "--parse", metavar="LOCATION", dest="location", choices=(
+    group: argparse._MutuallyExclusiveGroup = parser.add_mutually_exclusive_group(  # pylint: disable=protected-access
+        required=True,
+    )
+    group.add_argument(
+        "-p",
+        "--parse",
+        metavar="LOCATION",
+        dest="location",
+        choices=(
             ["fmi-bistro", "ipp-bistro", "mediziner-mensa"]
-            + list(menu_parser.StudentenwerkMenuParser.location_id_mapping.keys())),
-                        help="the location you want to eat at")
-    parseGroup: argparse._MutuallyExclusiveGroup = group.add_argument_group("parse")
+            + list(menu_parser.StudentenwerkMenuParser.location_id_mapping.keys())
```

Comment:
```suggestion
            + list(StudentenwerkMenuParser.location_id_mapping.keys())
```

Perhaps needs to be fixed for all following ones.